docs/ignite_cli/validator.md

```diff
@@ -0,0 +1,151 @@
+# Validator CLI Guide
+
+Validators are Cosmos SDK blockchain validators using `spn`
+
+---
+
+## List all published chains
+
+```shell
+ignite n chain list
+```
+
+### Output
+
+```shell
```

Comment:
We can remove all `shell` formatting for output, it gives wrong coloration sometimes